session-replacement/src/test/java/com/amadeus/session/servlet/TestWebXmlParser.java

```diff
@@ -55,7 +55,7 @@
       + "<web-app xmlns=\"http://java.sun.com/xml/ns/javaee\" "
       + "xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" "
       + "version=\"3.0\" xsi:schemaLocation=\"http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd\">"
-      + "<session-config><cookie-config><secure>false</secure><http-only>true</http-only></cookie-config>"
+      + "<session-config><cookie-config><secure>false</secure><http-only>true</http-only><path>/test</path></cookie-config>"
```

Comment:
Old bug here: to make web.xml from test correct, schemaLocation should point to 3.0: http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd